### PR TITLE
Fix incomplete transaction list in mw and dw, dw select dropdowns, and low-hanging fruits

### PR DESCRIPTION
**Pull request by nop33**
- Closes https://github.com/alephium/alephium-frontend/issues/137
- Closes https://github.com/alephium/alephium-frontend/issues/180
- Closes https://github.com/alephium/alephium-frontend/issues/178
- Closes https://github.com/alephium/alephium-frontend/issues/182
- Closes https://github.com/alephium/alephium-frontend/issues/177
- Closes https://github.com/alephium/alephium-frontend/issues/106

### CODE REVIEW

File: apps/mobile-wallet/src/contexts/walletConnect/WalletConnectContext.tsx [hunk 1]
```diff
@@ -334,7 +334,8 @@ export const WalletConnectContextProvider = ({ children }: { children: ReactNode
             autoHide: false
           })
         } else {
-          showExceptionToast(e, 'Could not build transaction')
+          if (!['alph_requestNodeApi', 'alph_requestExplorerApi'].includes(requestEvent.params.request.method))
+            showExceptionToast(e, 'Could not build transaction')
```
Comment (nop33) [inline apps/mobile-wallet/src/contexts/walletConnect/WalletConnectContext.tsx:338]: There is no point showing this error to the user. The user only cares about errors regarding signing.

**Comment by mvaivre** [comment]
Code looks good! Just a few comments regarding the selects' drop-down:

image

For simple options with one line, I'd keep the previous display using just the blue checkmark (no border)

image

There's an issue in the address select, the checkmark icon is visible even when the option isn't selected.

**Comment by nop33** [comment]
@mvaivre Your comments were addressed!

**Review by mvaivre — APPROVED** [review]
GJ! LGTM! 👏

**Pull request merged**